src/main/kotlin/com/cognifide/gradle/aem/base/vlt/VltCleaner.kt

```diff
@@ -135,6 +149,24 @@ class VltCleaner(val project: Project) {
         }
     }
 
+    private fun flattenFiles(root: File) {
+        if (root.isDirectory) {
+            allFiles(root).forEach { flattenFile(it) }
+        } else {
+            flattenFile(root)
+        }
+    }
+
+    private fun flattenFile(file: File) {
+        if (!file.exists() || !matchAnyRule(file.path, file, filesFlattenedRules)) {
+            return
+        }
+
+        logger.info("Flattening file {}", file.path)
+        val dest = File(file.parentFile.path + ".xml")
+        file.renameTo(dest)
```

Comment:
What if dest already exist? For instance when running only clean task without preceeding checkout?